Formula/tree-sitter.rb

```diff
@@ -15,10 +17,21 @@ class TreeSitter < Formula
     sha256 "fbef61fe89cc07c5af05a06185ccb91cfa9c34773ab6b26b751ae0171e3806a2" => :mojave
   end
 
+  depends_on "emscripten" => [:build, :test]
+  depends_on "node" => [:build, :test]
   depends_on "rust" => :build
-  depends_on "node" => :test
 
   def install
+    cd "lib/binding_web" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+    end
+    system "script/build-wasm"
+    mkdir_p lib/"binding_web"
+    cp "lib/binding_web/tree-sitter.js", lib/"binding_web/"
+    cp "lib/binding_web/tree-sitter-web.d.ts", lib/"binding_web/"
+    cp "lib/binding_web/tree-sitter.wasm", lib/"binding_web/"
+    cp "lib/binding_web/package.json", lib/"binding_web/"
```

Comment:
```suggestion
    %w[tree-sitter.js tree-sitter-web.d.ts tree-sitter.wasm package.json]. each do |file|
      (lib/"binding_web").install "lib/binding_web/#{file}"
    end
```

I think it's also better to move the installation of the wasm module to the end of the `install` block, rather than the beginning.

Reply:
Whoops, sorry for the syntax error.